server/cmd/mmctl/docs/mmctl_compliance_export_create.rst

```diff
@@ -0,0 +1,56 @@
+.. _mmctl_compliance_export_create:
+
+mmctl compliance_export create
+------------------------------
+
+Create a compliance export job, of type 'csv' or 'actiance' or 'globalrelay'
+
+Synopsis
+~~~~~~~~
+
+
+Create a compliance export job, of type 'csv' or 'actiance' or 'globalrelay'. If --date is set, the job will run for one day, from 12am to 12am (minus one millisecond) inclusively, in the format with timezone offset: `"YYYY-MM-DD -0000"`. E.g., "2024-10-21 -0400" for Oct 21, 2024 EDT timezone. "2023-11-01 +0000" for Nov 01, 2024 UTC. If set, the 'start' and 'end' flags will be ignored.
+
+Important: Running a compliance export job from mmctl will NOT affect the next scheduled job's batch_start_time. This means that if you run a compliance export job from mmctl, the next scheduled job will run from the batch_end_time of the previous scheduled job, as usual.
+
+::
+
+  mmctl compliance_export create [complianceExportType] --date "2025-03-27 -0400" [flags]
+
+Examples
+~~~~~~~~
+
+::
+
+  compliance_export create csv --date "2025-03-27 -0400"
+
+Options
+~~~~~~~
+
+::
+
+      --date "YYYY-MM-DD -0000"   Run the export for one day, from 12am to 12am (minus one millisecond) inclusively, in the format with timezoneoffset: "YYYY-MM-DD -0000". E.g., "2024-10-21 -0400" for Oct 21, 2024 EDT timezone. "2023-11-01 +0000"for Nov 01, 2024 UTC. If set, the 'start' and 'end' flags will be ignored.
```

Comment:
This multi-line split approach earlier has caused "timezone" and "offset" to collapse into a single word. Can we do something to fix this?